src/rust/engine/process_executor/src/main.rs

```diff
@@ -60,7 +62,13 @@ If unspecified, local execution will be performed.",
   };
   let server = args.value_of("server");
 
-  let request = process_execution::ExecuteProcessRequest { argv, env };
+  // TODO: Capture the working directory as a snapshot, and materialize it to a temporary directory.
```

Comment:
Would that be used somewhere? I'm not sure where... Seems more likely that we'd want to take a (list) of input snapshots as argument (depending on the snapshot manipulation API)? Either way, not important now.